docs/app/pages/components/components-sections/input-controls/typeahead/typeahead.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { IPlayground } from '../../../../../interfaces/IPlayground';
+import { IPlaygroundProvider } from '../../../../../interfaces/IPlaygroundProvider';
+import 'chance';
+import { DOCUMENTATION_TOKEN, DocumentationType } from '../../../../../services/playground/tokens/documentation.token';
+import { TypeaheadKeyService } from '@ux-aspects/ux-aspects';
+import { of } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Component({
+    selector: 'uxd-components-typeahead',
+    templateUrl: 'typeahead.component.html',
+    styleUrls: ['./typeahead.component.less'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
Can this be removed? Looks like there is only one style in the stylesheet that only effects this component so I don't think this is needed anymore.